internal/infrastructure/kubernetes/deployment.go

```diff
@@ -50,6 +50,13 @@ const (
 	envoyAdminPort = 19000
 	// envoyAdminAccessLogPath is the path used to expose admin access log.
 	envoyAdminAccessLogPath = "/dev/null"
+
+	// rateLimitInfraName is the name for rate-limit resources.
+	rateLimitInfraName = "envoy-ratelimit"
+	// rateLimitInfraHTTPPort is the http port that the rate limit service listens on.
+	rateLimitInfraHTTPPort = 8080
+	// rateLimitInfraImage is the container image for the rate limit service.
+	rateLimitInfraImage = "envoyproxy/ratelimit:latest"
```

Comment:
make this configurable in `EnvoyGateway.RateLimit`

Reply:
sure, rather than add a TODO comment in code, should be raised as a GH issue explaining the use cases where this is needed